src/public/WRLDRouteStep.h

```diff
@@ -70,6 +70,11 @@ NS_ASSUME_NONNULL_BEGIN
  */
 - (CLLocationDistance) distance;
 
+/*!
+ @returns name of current step
```

Comment:
Can this be updated to match rest of the comments. This is going to be showing up on our docs page (https://www.wrld3d.com/ios/latest/docs/api/Classes/WRLDRouteStep.html) so a better comment describing what this is, is extremely important.

Can you give me some details of what this variable contains. I think it contains the location name for this step but would like it confirmed before we decide on the docs comment.

Reply:
Yes, it contains step location name e.g "WRLD Office".

Reply:
Can we update it to `@returns If available, the name of the location this step is on.`

Reply:
fixed